stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -1519,7 +1523,7 @@ extension ${Self} {
 %   srcBits = src_type.bits
 %   That = src_type.stdlib_name
 
-%   if srcBits == 80:
+%   if (srcBits == 80) and (bits != 80):
```

Comment:
I don't think that this piece of the change does anything; what is your goal here?

Reply:
I was just syntactically annoyed that the existing implementation has:
#if !os(windows) && (arch(i386) || arch(x86_64))
//code
#if !os(windows) && (arch(i386) || arch(x86_64))
//code
#endif
#endif

Reply:
Hmmm, ok. Generally best to skip this sort of change-that-doesn't-really-change-anything, but it is a modest improvement and you've already done it, so let's take it.